src/spec/integration/orders-sync.spec.coffee

```diff
@@ -212,6 +222,70 @@ describe 'Integration Orders Sync', ->
       done()
     .catch (error) -> done(_.prettify(error))
 
+  it 'should sync line items', (done) ->
+    orderNew = _.deepClone @order
+    
+    orderNew.lineItems[0].state = [
+      {
+        quantity: 1,
+        fromState: {
```

Comment:
Why are you doing this? This means that when I use the SDK to generate update actions for line item states, I have to provide the `fromState` and `toState`. This seems wrong. We can just look ad the "old" order to get the from state and at the "new" order to get the new state, right?